src/controllers/PermissionController.js

```diff
@@ -0,0 +1,29 @@
+import dotenv from 'dotenv';
+import { User } from '../queries';
+// import * as validate from '../helpers/validation';
+import status from '../config/status';
+
+dotenv.config();
+/**
+ * A class to handle user local authentication
+ */
+export default class PermissionController {
+  /**
+   * @param {object} req
+   * @param {object} res
+   * @return {object} user information & token
+   */
+  static async findOrCreate(req, res) {
```

Comment:
I use findOrCreate to not create the permissions multiple times, if permissions are already created in the db it will no longer create it again. I do it once.

Reply:
Yes here i search if the permission exist if not i create it.